### PR TITLE
Indicate to user a new vsc def is required

### PR DESCRIPTION
**Pull request by weshayutin**
Consultant pointed out these docs are not quite clear. We need to indicate a new vsc must be created.

### CODE REVIEW

**Review by shubham-pampattiwar — COMMENTED** [review]
We can always annotate and add labels to vs class but cannot change the spec I believe

File: docs/examples/CSI/csi_example.md [hunk 1]
```diff
@@ -57,6 +57,8 @@
 
 ### Create a StorageClass and VolumeShapshotClass:
 
+- Exisiting VolumeSnapshotClass definitions can not be modified, a new VolumeSnapshotClass must be created.
```
Comment (kaovilai) [inline docs/examples/CSI/csi_example.md:60]: Only true for ones managed by an operator/controller.. which is most of cloud provider defaults

Comment (weshayutin) [reply docs/examples/CSI/csi_example.md:60]: thanks, updated!

File: docs/examples/CSI/csi_example.md [hunk 2]
```diff
@@ -60,6 +60,12 @@
 - A `StorageClass` and a `VolumeSnapshotClass` are needed before the Mssql application
```
[the remark this answers is not in the document]
Comment (weshayutin) [reply docs/examples/CSI/csi_example.md]: addressed in latest push, still self reviewing the entire doc.  Thank you!

**Review by shubham-pampattiwar — APPROVED** [review]

**Pull request merged**